lib/js-api/src/api/gs-core-rest-api/api/legal-hold.api.ts

```diff
@@ -52,4 +53,64 @@ export class LegalHoldApi extends BaseApi {
             returnType: HoldPaging
         });
     }
+
+    /**
+     * Assign hold of legal holds
+     *
+     * @param holdId The identifier of a hold.
+     * @param ids one element list with id of hold to assign to existing hold
+     * @returns Promise<HoldEntry>
+     */
+    assignHold(ids: [string], holdId: string): Promise<HoldEntry> {
```

Comment:
could you please rename id for mo specific, it this `holdId`, or `nodeId`?
```suggestion
    assignHold(ids: string[], holdId: string): Promise<HoldEntry> {
```